tutorials/tutorial-hello-timescale.md

```diff
@@ -0,0 +1,453 @@
+ 1         | 2016-01-01 00:00:01 | 2016-01-01 00:11:55 |               1 |          1.20 | -73.979423522949219 | 40.744613647460938 |         1 | -73.992034912109375 | 40.753944396972656 |            2 |           9 |   0.5 |     0.5 |          0 |            0 |                   0.3 |         10.3
+ 1         | 2016-01-01 00:00:02 | 2016-01-01 00:11:14 |               1 |          6.00 | -73.947151184082031 | 40.791046142578125 |         1 | -73.920768737792969 | 40.865577697753906 |            2 |          18 |   0.5 |     0.5 |          0 |            0 |                   0.3 |         19.3
+ 1         | 2016-01-01 00:00:07 | 2016-01-01 00:09:49 |               1 |          1.80 | -73.989166259765625 | 40.726589202880859 |         1 | -74.009483337402344 | 40.715072631835938 |            2 |           9 |   0.5 |     0.5 |          0 |            0 |                   0.3 |         10.3
+ 1         | 2016-01-01 00:00:09 | 2016-01-01 00:07:18 |               2 |          1.20 | -73.963912963867188 | 40.712173461914062 |         1 | -73.951332092285156 | 40.712200164794922 |            2 |           7 |   0.5 |     0.5 |          0 |            0 |                   0.3 |          8.3
+ 1         | 2016-01-01 00:00:16 | 2016-01-01 00:33:00 |               2 |          2.90 | -73.982154846191406 | 40.774879455566406 |         1 | -73.981361389160156 | 40.744113922119141 |            2 |        20.5 |   0.5 |     0.5 |          0 |            0 |                   0.3 |         21.8
+(5 rows)
+
+## Mission 2: Analysis
+
+<img class="main-content__illustration" style="width: 100%" src="https://s3.amazonaws.com/docs.timescale.com/hello-timescale/NYC_picture_3.png" alt="NYC Taxis"/>
+
+The day is 1 February 2016, and the government has just issued a notice that, in an effort to mitigate the impact of global warming, all public entities must take measures to decrease their carbon footprint. New York City has committed to reducing their greenhouse gas emissions by 20% by 2024. Given the number of taxi rides taken each day, they believe studying past taxi rider history will play a major factor in doing so.
+
+The New York Data Team have recruited you to help them make sense of historical taxi ride data, draw insights, and use your analysis to plan for the future. Your goal is to conduct an analysis of all NYC TLC taxi rides taken  in January 2016.
+
+### 2.1 Basic Analysis 
+
+#### 2.1.1 How many rides took place on each day?
+
+The first question that they’ve asked you to explore is simple: How many rides took place on each day during January 2016?
+
+Since TimescaleDB supports full SQL, all that’s required is a simple SQL query to count the number of rides and group/ order them by the day they took place, as seen in query 2.1.1 below:
```

Comment:
```suggestion
Since TimescaleDB supports full SQL, all that’s required is a simple SQL query to count the number of rides and group/ order them by the day they took place:
```
It's implied that the immediately following code block is related to this paragraph.  Same deal for all of the other examples.